pom.xml

```diff
@@ -147,7 +147,7 @@
         <dependency>
             <groupId>com.jupiter-tools</groupId>
             <artifactId>compile-test</artifactId>
-            <version>0.1</version>
+            <version>0.2</version>
```

Comment:
@antkorwin so now that compile-test v0.2 is on Maven Central (:tada:), could you re-run the most recent test? I apparently can't trigger a test re-run on Travis for this project myself. Or I can just amend & force-push the most recent commit.